apps/vs-code-data-mapper/src/DataMapperPanel.ts

```diff
@@ -158,7 +158,10 @@ export default class DataMapperPanel {
     fs.readdir(path.join(DataMapperExt.getWorkspaceFolderFsPath(), schemasPath)).then((result) => {
       this.sendMsgToWebview({
         command: 'showAvailableSchemas',
-        data: result.filter((file) => path.extname(file).toLowerCase() === '.xsd'),
+        data: result.filter((file) => {
+          const fileExt = path.extname(file).toLowerCase();
+          return fileExt === '.xsd' || fileExt === '.json';
```

Comment:
Should this be hooked up to supportedSchemaFileExts too?

Reply:
Changed